### PR TITLE
docs(design): update getting started docs

### PR DESCRIPTION
**Pull request by xelaint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/develop/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Part of: #2767 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: libs/design/guides/getting-started.md [hunk 1]
```diff
@@ -1,96 +1,74 @@
...
 @forward '@daffodil/design/scss/global';
 ```
 
-> For more information on our approach to these kinds of styles, see the ["Global Styles" guide.](/libs/design/guides/global-styles.md)
+> For more information on our approach to these kinds of styles, see the [Global Styles guide.](/libs/design/guides/foundations/global-styles.md)
 
-### Add a Theme
-`@daffodil/design` is a themable component library. The components in the design library can be configured with customized colors in addition to a dark AND light mode for those same colors.
+## Add a theme
+`@daffodil/design` is a themable component library. The components in the design library can be configured with customized colors in addition to a dark and light mode for those same colors. No further configuration is needed if you choose to use `@daffodil/design`'s default theme.
```
Comment (damienwebdev) [inline libs/design/guides/getting-started.md]: Is this correct? Don't you still need to add the theme `@use`?

File: libs/design/guides/getting-started.md [hunk 2]
```diff
@@ -103,5 +81,5 @@ With the module imported, the `AppComponent` template can be updated to render t
 </daff-hero>
 ```
 
-## Next Steps
-We've just walked through the basics of setting up the `@daffodil/design` library. There is much more to it than just the `DaffHeroComponent`. Check out the full list of available components, try and add them to your sample app, and imagine all the wonderful things you can now build!
\ No newline at end of file
+## Next steps
```
Comment (damienwebdev) [inline libs/design/guides/getting-started.md:98]: Maybe leave a small challenge here? Something like "try to change the color of the hero" and link to the docs of the hero component?

**Review by damienwebdev — CHANGES_REQUESTED** [review]

**Review by gracetxgao — DISMISSED** [review]

File: libs/design/guides/getting-started.md (hunk 1 above)
Comment (xelaint) [reply libs/design/guides/getting-started.md]: updated

File: libs/design/guides/getting-started.md (hunk 2 above)
Comment (xelaint) [reply libs/design/guides/getting-started.md:98]: added

**Pull request merged**